BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -31,6 +31,7 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate,
     var onlyShowOpenStores = true
     let edgeGesture = UIScreenEdgePanGestureRecognizer()
     var panningWasTriggered = false
+    let defaults = SideMenuFilterViewController().defaultPreferences
```

Comment:
`UserDefaults` is a global class. It functions essentially like a Singleton, so you do not really need to steal it from `SideMenuFilterViewController`.